sync-ios-app/DataManager.swift

```diff
@@ -31,18 +31,65 @@ open class DataManager: NSObject {
         conf?.notifySyncCompleted = true
         conf?.notifySyncFailed = true
         conf?.notifyRemoteUpdateApplied = true
+        conf?.notifySyncCollision = true;
         conf?.notifyRemoteUpdateFailed = true
         conf?.notifyLocalUpdateApplied = true
         conf?.notifyDeltaReceived = true
         conf?.crashCountWait = 0;
         syncClient = FHSyncClient(config: conf)
         NotificationCenter.default.addObserver(self, selector:#selector(DataManager.onSyncMessage(_:)), name:Notification.Name(rawValue: "kFHSyncStateChangedNotification"), object:nil)
         syncClient.manage(withDataId: DATA_ID, andConfig:nil, andQuery:[:])
+        
     }
     
     @objc open func onSyncMessage(_ note: Notification) {
         if let msg = note.object as? FHSyncNotificationMessage, let code = msg.code {
             print("Got notification: \(msg)")
+            
+            if code == COLLISION_DETECTED_MESSAGE {
```

Comment:
It will be better if it is a switch. 

<img width="1073" alt="screen shot 2018-07-31 at 09 41 40" src="https://user-images.githubusercontent.com/7708031/43450355-762d1ffa-94aa-11e8-8b8f-4335722730ed.png">